api/errors.py

```diff
@@ -0,0 +1,15 @@
+from http import HTTPStatus
+
+
+class WrongCredentialsError(Exception):
+
+    def __init__(self):
+        super().__init__('Wrong credentials')
+        self.message = 'Wrong credentials'
+        self.status_code = HTTPStatus.BAD_REQUEST
+
+    def json(self):
+        return {
+            'status_code': self.status_code,
+            'message': self.message
+        }
```

Comment:
Please make a base class and inherit it here, because potentially you can have more than one custom error in the future